### PR TITLE
More release-please fixes

### PR DESCRIPTION
**Pull request by mdesouky**
### Changes:

- Fix initial version of CLI
- Update `appVersion` in helm chart with new server releases

### CODE REVIEW

File: release-please-config.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
         "extra-files": [
```
Comment (uptickmetachu) [inline release-please-config.json:17]: Maybe we just exclude the helm charts for updates to gitops serer.

**Review by uptickmetachu — APPROVED** [review]
Approving for flow. You can also run release-please locally if you generate a GITHUB_TOKEN.

File: release-please-config.json (hunk 1 above)
Comment (mdesouky) [reply release-please-config.json:17]: let's try it first, if it doesn't work, then sure.

**Comment by mdesouky** [comment]
> Approving for flow. You can also run release-please locally if you generate a GITHUB_TOKEN.

@uptickmetachu I'll try running it locally first then, if it does what it's supposed to, I'll merge.

**Pull request merged**